api/WebGLObject.json

```diff
@@ -20,7 +20,7 @@
             "version_added": null
           },
           "ie": {
-            "version_added": null
+            "version_added": "11"
```

Comment:
Interesting. I guess it was only IE/Edge that had this base interface for `WebGLBuffer` and other interfaces. The Edge data will also need updating, since I assume it's not in new Edge.